src/components/molecules/MessageFormatting/MessageFormatting.tsx

```diff
@@ -7,6 +7,226 @@ import styles from './MessageFormatting.module.scss';
 
 const messageFormattingContent = content.components.messageFormattingComponent;
 
+const LineBreaksAndParagraphs = () => (
```

Comment:
Nice refactor, you _could_ go a step further and put these into their own files.

Reply:
I can't take credit for this. This was taken from the changes made my @bhansell1

Reply:
step further captured in tidy up https://nhsd-jira.digital.nhs.uk/browse/CCM-6950